src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1688,6 +1691,73 @@ Expression* TranslateToFuzzReader::makeTry(Type type) {
   return builder.makeTry(body, catchTags, catchBodies);
 }
 
+Expression* TranslateToFuzzReader::makeTryTable(Type type) {
+  if (funcContext->breakableStack.empty()) {
+    // Nothing to break to.
+    // TODO: Perhaps generate a block wrapping us?
+    return makeTrivial(type);
+  }
```

Comment:
We can actually make a trivial `try_table` that doesn't need branches (not sure how much it would help fuzzing though)
```wast
try_table (result type)
  ...
end_try_table
```

Reply:
Oh, good point. I think that is better, as it could catch something. It also makes the code simpler. I updated to do that now.